contrib/runners/local_runner/local_runner/runner.yaml

```diff
@@ -33,6 +33,24 @@
       description: Action timeout in seconds. Action will get killed if it doesn't
         finish in timeout seconds.
       type: integer
+  output_key: stdout
+  output_schema:
+    type: object
+    properties:
+      succeeded:
+        type: boolean
+      failed:
+        type: boolean
+      stdout:
+        anyOf:
+          - type: "object"
+          - type: "string"
+          - type: "array"
+          - type: "number"
```

Comment:
Can this be any of the simple json types? Why these 4?
https://github.com/StackStorm/st2/blob/master/st2common/st2common/util/schema/action_params.json#L19

I guess we need to look at how stdout is converted from json to see if there are any restrictions.